core/src/main/scala/org/apache/spark/ExecutorAllocationManager.scala

```diff
@@ -101,6 +104,11 @@ private[spark] class ExecutorAllocationManager(
   private val executorIdleTimeoutS = conf.getTimeAsSeconds(
     "spark.dynamicAllocation.executorIdleTimeout", "60s")
 
+  private val cachedExecutorTimeoutS = conf.getTimeAsSeconds(
+    "spark.dynamicAllocation.cachedExecutorIdleTimeout", s"${Long.MaxValue / 1000}s")
```

Comment:
if we're using `Long.MaxValue` it probably doesn't matter if we divide by a 1000, but this is OK to keep :)